apps/www/components/Audio/AudioPlayer/ui/AudioCover.tsx

```diff
@@ -27,6 +27,10 @@ const styles = {
 const getResizefromURL = (url, size) => {
   const imgURL = new URL(url)
   const sizeString = imgURL.searchParams.get('size')
+  if (!sizeString) {
```

Comment:
Before this was added, this function caused an exception if the size query-param was not given. This seemed like the right way to handle that case.